servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLSchemaConfiguration.kt

```diff
@@ -159,4 +160,18 @@ class GraphQLSchemaConfiguration {
         contextFactory,
         requestHandler
     )
+
+    @Bean
+    @ConditionalOnMissingBean
+    fun schemaPrinter(): SchemaPrinter {
+        return SchemaPrinter(
+            SchemaPrinter.Options.defaultOptions()
+                .includeIntrospectionTypes(false)
+                .includeScalarTypes(true)
+                .includeSchemaDefinition(true)
+                .includeDirectives(true)
+                .includeDirectives { true }
+                .includeDirectiveDefinitions(true)
+        )
```

Comment:
The values here all match the defaults from [GraphQLSchema.print()](https://github.com/ExpediaGroup/graphql-kotlin/blob/095b056b74d52237b73cef872fdbd851c019039c/generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/extensions/GraphQLSchemaExtensions.kt#L34-L51) 

I'd be open to suggestions for a good place to codify the shared set of defaults, and use that instead of defining a duplicate set of values here.